addons/crm/tests/common.py

```diff
@@ -417,13 +417,21 @@ def _aggregate(fname):
                 values = [_find_value(lead, fname) for lead in leads]
             return values
 
+        def _get_address(fname):
+            for lead in leads:
```

Comment:
Shouldn't we explicitly give the "address valid" lead in assert to ensure the right one is chosen ? I fear we may have bugs hidden in an heuristic in a test ...

Reply:
We can just ignore the address field and let us give the value manually (like we already do)

Reply:
Probably easier indeed :)